src/main/cpp/CMakeSettings.json

```diff
@@ -0,0 +1,49 @@
+{
+  // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
```

Comment:
`"_comment": "See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.",`